linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml

```diff
@@ -26,6 +26,10 @@ ocil_clause: |-
     <pre>include "/etc/crypto-policies/back-ends/bind.config";</pre> directive
 
 ocil: |-
+    Either BIND is not installed, or it is configured to use the system-wide defined crypto policy.
+
+    {{{ ocil_package(package="bind") }}}
+
```

Comment:
This should be in the OCIL clause and `{{{ ocil_package(package="bind") }}}` is not needed.

Reply:
Edited and moved to OCIL clause. Hope it makes sense.